server/db/models/ordersproducts.js

```diff
@@ -19,7 +20,16 @@ const OrdersProducts = db.define('ordersProducts', {
     defaultValue: null,
   },
 })
-
+OrdersProducts.prototype.getTotalPrice = async function() {
+  let thisPrice = this.purchasePrice
+  if (thisPrice === null) {
+    let chosenProduct = await Axios.get(`/api/products/${this.productId}`)
```

Comment:
Any reason we can't just import the Products model and query it directly, here, instead of having the server hit up one of its own API routes?